tests/lang_tests_common.rs

```diff
@@ -115,7 +115,19 @@ pub fn main_inner(profile: Profile) {
                 }
             }
             match profile {
-                Profile::Debug => {}
+                Profile::Debug => {
+                    if test_target.is_ok() {
+                        // m68k doesn't have lubsan for now
+                        compiler.args(["-C", "llvm-args=sanitize-undefined"]);
+                    } else {
+                        compiler.args([
+                            "-C",
+                            "llvm-args=sanitize-undefined",
```

Comment:
Are you sending the correct flags here?
I ran the new test on the master branch and it passes while I assumed it should fail on the master branch.